kbase-extension/static/kbase/js/util/appCellUtil.js

```diff
@@ -184,9 +188,25 @@ define(['common/runtime', 'StagingServiceClient'], (Runtime, StagingServiceClien
             });
     }
 
+    function generateFileTypeMappings(typesToFiles = {}) {
```

Comment:
add this function -- it's used in both the BI cell's configure tab  and will be used in the job status table